### PR TITLE
Feature/choose goprod

### PR DESCRIPTION
**Pull request by ChemiKyle**
Address issue #22

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -54,6 +54,21 @@
             "required": false,
             "type": "textarea",
             "repeatable": false
+        },
+        {
+            "key": "use_goprod",
+            "name": "Use goprod module?",
+            "type": "radio",
```
Comment (ChemiKyle) [inline config.json:61]: Interesting note: making this a checkbox with only the "Yes" option did not actually allow toggling

File: config.json [hunk 2]
```diff
@@ -54,6 +54,21 @@
...
+            "choices": [
+                {
+                    "name": "No",
+                    "value": ""
```
Comment (ChemiKyle) [inline config.json:65]: Less interestingly, setting this value to null makes it appear that no choice is set when the user reopens the configuration

**Comment by marlycormar** [comment]
Hi @ChemiKyle, in commit f7f1077 you modify the sql query by adding `AND redcap_projects.date_deleted IS NULL`. Why is this needed?

**Comment by marlycormar** [comment]
There are two issues with the PR:
1. The external module seems not to be using the go_prod module or plugin as having both disabled has no effect in the module. **Update**: this is true but ignoring it since the module requires go_prod to be enabled.
~2. If the module is enabled and then the module version is changed to an older one, the table shown in the link 'Production Candidates Report' still displays the column with text 'Go to Prod'.~ **Update**: go_prod was used by default in previous versions. 

RPC

**Pull request merged**